index.html

```diff
@@ -1,9 +1,35 @@
 <!DOCTYPE html>
 <html>
-<head>
-<title>Fi+</title>
-</head>
-<body>
-<h1>Hello World!</h1>
-</body>
+	<head>
+		<title>The Fi+ Android App<title>
+	</head>	
+	
+	<body>
+		<h1>Welcome to the Fi+ Android App!</h1>
```

Comment:
Look into sections. Would be useful for when we want to incorporate JS so that not all the text shows up at once, but all the text shows up.